docs/source/software/spack.md

```diff
@@ -45,6 +55,8 @@ This section details how to install the LAMMPS application with the KOKKOS and U
    srun --partition=gpu --nodes=1 --ntasks=14 --pty --export=All --gres=gpu:1 --mem=0 --time=08:00:00 /bin/bash
    :::
 
+1. Load a compatible cuda module: `module load cuda/10.2`
+
 1. (Optional) Initiate a `screen` session:
```

Comment:
As a group, are we backing `screen` in our documentation or `tmux`? Just for consistency we probably should decide upon one and stay with it.